test/spec/ng-infinite-scroll.spec.js

```diff
@@ -37,7 +47,7 @@ const getTemplate = (angularVersion, container, attrs, throttle) =>
       .run(function ($rootScope) {
         $rootScope.items = [];
         $rootScope.loadMore = function () {
-          [].push.apply($rootScope.items, new Array(100));
+          [].push.apply($rootScope.items, new Array(${nElements}));
```

Comment:
I'm trying to be consistent with the rest of the template. Care to explain your suggestion?